### ISSUE TITLE
Pushy.register() fails attempting to use fetch

### ISSUE THREAD

**Issue by chasegiunta** [pushy-me/pushy-electron#2]
```
Error: The API request failed: fetch is not a function
    at Object.register (webpack:///./node_modules/pushy-electron/lib/Pushy.js?:81:19)
```

**Comment by pushy-me** [comment]
Hi @chasegiunta,
For `fetch` support in Electron, we're using a dependency `node-fetch`:
https://www.npmjs.com/package/node-fetch

Seems like your webpack setup is having a hard time processing this dependency. Can you check for any errors arising in webpack?

**Comment by chasegiunta** [comment]
Webpack doesn't provide further errors, but it looks like the `node-fetch` repo is full of issues pertaining to webpack. This seemed to be the most straightforward approach:
https://github.com/node-fetch/node-fetch/issues/450#issuecomment-387045223

So replacing this line
https://github.com/pushy-me/pushy-electron/blob/1620dbde5f2635350cf10e43131b9cbea78c800e/util/api.js#L1
with `const fetch = require('node-fetch').default` resolves the issue for me locally, though I'm not sure if there are any other unintended side-effects in non-webpack environments.

Also, looks like version 3 of node-fetch, current in beta, may already have fixed this, but for the time being would it be possible to add `.default` to the require?

**Comment by pushy-me** [comment]
Excellent workaround @chasegiunta, tested and works without webpack as well! 🎉 

Committed in https://github.com/pushy-me/pushy-electron/commit/b8c07603bfd8950dfd21155e0274e98290c0afb7 and published to `npm` in `1.0.5`. Thanks for your contribution!
